### ISSUE TITLE
Question about learning rate

### ISSUE THREAD

**Issue by DUT-CSJ** [juhongm999/hsnet#29]
Hello, thank you for sharing your work!
When I try to reproduce hsnet, I find the learning rate is fixed through the training.
I trained hsnet on pascal VOC2012 (1-shot, fold0) about 400 epochs, the miou on val set is very unstable (highest: 63.62 lower than 64.3 in paper). (gpu: single 3090, other settings are same to default.)
Dose the lr need to be adjust?
Looking forward to your reply.

**Comment by juhongm999** [comment]
For accurate reproduction of our results, you should follow the same training settings as the ones we used. Specifically, our network was trained with a batch size of 20 and four Titan RTX GPUs over a couple of days. Please aim to replicate these settings as closely as possible.

**Comment by DUT-CSJ** [comment]
> For accurate reproduction of our results, you should follow the same training settings as the ones we used. Specifically, our network was trained with a batch size of 20 and four Titan RTX GPUs over a couple of days. Please aim to replicate these settings as closely as possible.

I don't have the same gpu with yours, what else, thank you for your reply!
